tools/ovc/openvino/tools/ovc/convert_impl.py

```diff
@@ -76,20 +77,38 @@ def print_argv(argv: argparse.Namespace):
     print('\n'.join(lines), flush=True)
 
 
+def check_iterable_types(iterable: Iterable, type_name: type):
+    for element in iterable:
+        if not isinstance(element, type_name):
+            return False
+    return True
+
+
 def arguments_post_parsing(argv: argparse.Namespace):
     # TODO: This function looks similar to another one. Check for code duplicates.
     log.debug("Model Conversion API started")
     if not argv.is_python_api_used:
         log.debug('Output model name would be {}{{.xml, .bin}}'.format(argv.output_model))
 
-    if argv.verbose:
+    if is_verbose(argv):
         print_argv(argv)
 
     params_parsing(argv)
-    argv.output = argv.output.split(',') if isinstance(argv.output, (str, pathlib.Path)) else argv.output
-
     log.debug("Placeholder shapes : {}".format(argv.placeholder_shapes))
 
+    if not hasattr(argv, 'output') or argv.output is None:
+        return argv
+
+    if argv.is_python_api_used:
+        error_msg = f"output '{argv.output}' is incorrect, it should be string or a list/tuple of strings"
+        assert isinstance(argv.output, (str, list, tuple)), error_msg
+        if isinstance(argv.output, list):
+            assert check_iterable_types(argv.output, str), error_msg
+        else:
+            argv.output = [argv.output]
+    else:
+        assert isinstance(argv.output, str)
+        argv.output = argv.output.replace(' ', '').split(',')
```

Comment:
This is not good. If the space is inside the name and there is no comma nearby it means the wrong name. For example: "na me" is not the same as "name", this is an error. Use `regex.split(r'\s*,\s*', argv.output.strip())` and check that the resulting elements don't contain spaces and are not empty, or raise an error with the explanation otherwise.

Reply:
done